### PR TITLE
Adding comments to plotting functions

### PR DESCRIPTION
**Pull request by arahuja**
This add some docstrings to the two main plotting functions in `load.py`

Also, sets the `xlabel` for KM plots.

### CODE REVIEW

File: cohorts/load.py [hunk 1]
```diff
@@ -307,7 +304,28 @@ def plot_init(self, on, col, col_equals):
...
+        odds-ratios and perform a Fisher's exact test.
+
+        If the variable is numeric, this will compare the distributions through
+        a Mann-Whitney test and plot the distributions with box-strip plot
+
+        Parameters
+        ----------
+        on : See `cohort.load.plot_init`
+        col : str, optional
+            If `on` is not specified, the column name on which to split the cohort
```
Comment (tavinathanson) [inline cohorts/load.py]: This API is even confusing me already, so clearly could use improvement! 

That said, `on` is required, so this comment looks incorrect?

Comment (arahuja) [inline cohorts/load.py]: Oh ok, I think I needed to read more of the `plot_init` docs, but I think I get it now. We are going to `plot_init` no matter what and `col` if it'll be a new column created from `on` based on `col_equals`?

Comment (tavinathanson) [inline cohorts/load.py]: Yeah, basically; or `on` can just be a string name for a new column generated by comparing `col` with `col_equals`

**Comment by tavinathanson** [comment]
Thanks! Looks good % minor comment

File: cohorts/load.py (hunk 1 above)
Comment (arahuja) [inline cohorts/load.py]: Hm, in the end it was a bit hard to succinctly state what `col` was. Gave it an attempt, but if you want we can just leave it as "Refer to `plot_init`"

**Comment by tavinathanson** [comment]
LGTM :+1:

**Pull request merged**